.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: node:latest
+      - image: node:8.8
```

Comment:
8.9?

Reply:
unfortunately not, `npm login` is broken